### ISSUE TITLE
Use custom `selectedWCs.txt` file instead of static committed file

### ISSUE THREAD

**Issue by kmohrman** [TopEFT/EFTFit#18]
In the current implementation, the `selectedWCs.txt` used in the t2w step is committed to the repo. This can be problematic if the committed file does not match the actual selected WCs (selected in the data card making step). There are two possible consequences, the first of which is relatively benign and the second of which can be very problematic: 
1. If the committed list is smaller than the actual selected list, t2w will raise an error something like the following: 
    ```
    RuntimeError: Cannot add non-existant factor func_ttlnu_sm_linear_quadratic_mixed_ctei_ctq8 for process ttlnu_quad_mixed_ctq8_ctei, bin ch1
    ```
    If you get this error, you know that the selected WC lists to not match, and you have to swap out the committed one for the one generated in the datacard making step. This can be annoying, but it is relatively benign since you are unable to proceed with the wrong file. 
2.  If the committed list is larger than the actual selected list, an error will not be raised. The t2w step and the subsequent fitting steps will run without errors. However, the results will be wrong. This pernicious issue has impacted us multiple times throughout TOP-22-006. 

For these reasons (mainly Reason 2), it would be much safer to reimplement so that there is not a committed static `selectedWCs.txt` file to begin with. Rather, the user would be expected to copy the `selectedWCs.txt` file (produced in the datacard making step) into the same directory as the card/templates when preparing to run the t2w step. The physics model would then look to the `selectedWCs.txt` in that local directory instead of looking to the committed `selectedWCs.txt` file.

**Comment by hannahbnelson** [comment]
I've made a pull request that fixes this, [PR #22](https://github.com/TopEFT/EFTFit/pull/22). 
The changes are in the EFTFit repository in commit [e31f923](https://github.com/TopEFT/EFTFit/commit/e31f923c3a7a963c41f4a21697e30366603664e6)

**Comment by kmohrman** [comment]
Closed by PR #22 PR #23
